docs/userguide/boundaryanalysis/index.md

```diff
@@ -237,7 +237,7 @@ French, Italian and Portuguese.
 
 ICU's break iterators are based on the default boundary rules described in the
 Unicode Standard Annexes [14](https://www.unicode.org/reports/tr14/) and
-[29](https://www.unicode.org/unicode/reports/tr29/) . These are relatively
+[29](https://unicode.org/reports/tr29/) . These are relatively
```

Comment:
Hmm, it looks like both of these URLs work:
https://www.unicode.org/reports/tr29/
https://unicode.org/reports/tr29/

I wonder which might be better...? (I'm not sure TBH.) 
Though the link as-is doesn't work, so fixing it is a good catch!

Reply:
Let's keep *www* for consistency with other links. I just copied from browser without get full link (copy-paste exception 😅 ). 
Fixed.

Reply:
Sounds good, thanks!